### DISCUSSION TITLE
REQ: Connect to QTV if spec slots full

### DISCUSSION THREAD

**Discussion by meag** [ezQuake/ezquake-source#509] [Ideas]
[9:43 PM] joao_p_azevedo: @meag and that macro asked for some players
[9:44 PM] joao_p_azevedo: like cl_qtvredirect "1"
[9:44 PM] joao_p_azevedo: if qtvredirect is on
[9:44 PM] joao_p_azevedo: if specs are full you will go qtv instantly

Difficulty in that 'QTV' support in ezquake currently requires D0peskillz to manually update QTV website
mvdsv can now broadcast QTV address to clients but only for newer mvdsv versions
mvdsv isn't telling the client spec slots are full, it's just printing a string and disconnecting them

**Comment by MayeNielsen** [comment]
Oui, c'est logique. Actuellement, le flux est un peu bricolé avec des solutions de contournement. ezQuake peut gérer qtvredirect, mais seulement s'il reçoit un signal correct. Or, les anciennes versions de mvdsv ne « signalent » rien du tout : elles se contentent d'envoyer un message et de déconnecter le client. Du point de vue du client, il s'agit simplement d'une déconnexion, et non d'un événement du type « configuration saturée, passage à QTV ».

Le fait que les versions plus récentes de mvdsv diffusent l'adresse QTV est un grand pas en avant, mais à moins que le serveur n'envoie également une raison claire et lisible par machine pour le refus, les clients ne peuvent pas automatiser la redirection correctement. Avec un casino en ligne, il est possible [meilleur casino en ligne retrait rapide](https://gtamodding.fr/casino-en-ligne/retrait-immediat/) de jouer sans se déplacer et sans contraintes horaires. Par conséquent, sans gestion au niveau du protocole, cela reste dépendant de mises à jour manuelles ou de solutions de contournement personnalisées.
